setup_build.py

```diff
@@ -195,12 +196,14 @@ def run(self):
 DEF VDS_MIN_HDF5_VERSION = (1,9,233)
 DEF VOL_MIN_HDF5_VERSION = (1,11,5)
 DEF COMPLEX256_SUPPORT = %(complex256_support)s
+DEF FREE_GIL = %(free_gil)s
 """
                 s %= {
                     'mpi': bool(config.mpi),
                     'mpi4py_v2': bool(v2),
                     'version': tuple(int(x) for x in config.hdf5_version.split('.')),
-                    'complex256_support': hasattr(numpy, 'complex256')
+                    'complex256_support': hasattr(numpy, 'complex256'),
+                    'free_gil': FREE_GIL
```

Comment:
This config option doesn't seem to be used anywhere.

Reply:
Indeed. James suggested to use conditional system wide gil deactivation based on constants, which looked like a cool idea.
Unfortunately this is not compatible with the "with gil" function declaration that automatically re-acquires the gil when needed.

Reply:
I removed this part as it is not used